addons/l10n_tz/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo auto_sequence="1">
+    <record id="tax_report" model="account.report">
+        <field name="name">Tax Report</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.tz"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="tax_report_value" model="account.report.column">
+                <field name="name">Value (Excluding VAT)</field>
+                <field name="expression_label">value</field>
+            </record>
+            <record id="tax_report_rate" model="account.report.column">
+                <field name="name">VAT Rate</field>
+                <field name="expression_label">rate</field>
+            </record>
+            <record id="tax_report_amount" model="account.report.column">
+                <field name="name">VAT Amount</field>
+                <field name="expression_label">amount</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="tax_report_goods_or_services" model="account.report.line">
+                <field name="name">Supplies of goods &amp; or Services</field>
+                <field name="code">tz_goods_or_services</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="tax_report_taxable_supplies" model="account.report.line">
+                        <field name="name">Taxable supplies</field>
+                        <field name="code">taxable_supplies</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_taxable_supplies_expr_value" model="account.report.expression">
+                                <field name="label">value</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">02.base</field>
+                            </record>
+                            <record id="tax_report_taxable_supplies_expr_rate" model="account.report.expression">
+                                <field name="label">rate</field>
+                                <field name="engine">external</field>
+                                <field name="formula">sum</field>
+                                <field name="subformula">editable</field>
+                            </record>
```

Comment:
It might just be me, but what is the purpose of this if it is not used?
Just for display?

Reply:
@william-andre I am not quite sure, but it was there in the reference tax report, and @Laura-labe requested it to be editable field.

Reply:
Hello, the government requires it in the tax report.
![2024-06-25_13-21](https://github.com/odoo/odoo/assets/118346429/e74167ae-6b5e-4180-a533-45f45f72b707)
In order to stay true to the original report, I wanted this column to be present.